mailpoet/tests/unit/WooCommerce/TransactionalEmails/FontFamilyValidatorTest.php

```
@@ -0,0 +1,156 @@
+  }
+
+  public function testValidateFontFamilyHandlesMaliciousInput() {
+    $maliciousInputs = [
+      'Arial" onload="alert(1)" x="' => 'Arial onload=alert(1) x=',
+      'Times" onclick="alert(\'XSS\')" data="' => 'Times onclick=alert(XSS) data=',
+      'Arial; background: url(javascript:alert(1));' => 'Arial background: url(javascript:alert(1))',
+      'Arial</style><script>alert(1)</script><style>' => 'Arialstylescriptalert(1)scriptstyle',
+      'Times"</style><script>alert(1)</script><style a="' => 'Timesstylescriptalert(1)scriptstyle a=',
+    ];
+
+    foreach ($maliciousInputs as $input => $expected) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$expected'. (undefined)

(UnusedLocalVariable)